### PR TITLE
Migrate active response scenarios

### PR DESCRIPTION
**Pull request by QU3B1M**
|Related issue|
|---|
| #17426 |

## Description

This issue applies the refactored `test_execd` and `test_analysisd\test_trigger_active_response` scenarios from the `/wazuh/wazuh-qa` repo. It also comes with new fixtures required for this test.

### CODE REVIEW

File: tests/integration/test_execd/test_run_active_response/conftest.py
Comment (TomasTurina) [inline tests/integration/test_execd/test_run_active_response/conftest.py:1]: Move what you consider to a general `conftest.py` in execd folder.

File: tests/integration/test_execd/test_run_active_response/test_firewall_drop.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+pytestmark = [pytest.mark.agent, pytest.mark.tier(level=1)]
+
+# Cases metadata and its ids.
+cases_path = Path(TEST_CASES_PATH, 'cases_execd_firewall_drop.yaml')
+config_path = Path(CONFIGS_PATH, 'config_run_active_response.yaml')
+test_configuration, test_metadata, cases_ids = get_test_cases_data(cases_path)
+test_configuration = load_configuration_template(config_path, test_configuration, test_metadata)
+
+# Test internal options and configurations.
+local_internal_options = EXECD_DEBUG_CONFIG
```
Comment (TomasTurina) [inline tests/integration/test_execd/test_run_active_response/test_firewall_drop.py:68]: Remember to change this to create the dict here.

File: tests/integration/test_execd/test_run_active_response/test_restart_wazuh.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+pytestmark = [pytest.mark.agent, pytest.mark.tier(level=1)]
+
+# Cases metadata and its ids.
+cases_path = Path(TEST_CASES_PATH, 'cases_execd_restart.yaml')
+config_path = Path(CONFIGS_PATH, 'config_run_active_response.yaml')
+test_configuration, test_metadata, cases_ids = get_test_cases_data(cases_path)
+test_configuration = load_configuration_template(config_path, test_configuration, test_metadata)
+
+# Test internal options and configurations.
+local_internal_options = EXECD_DEBUG_CONFIG
```
Comment (TomasTurina) [inline tests/integration/test_execd/test_run_active_response/test_restart_wazuh.py:73]: Same.

**Review by TomasTurina — CHANGES_REQUESTED** [review]

File: tests/integration/test_execd/test_run_active_response/test_firewall_drop.py (hunk 1 above)
Comment (QU3B1M) [reply tests/integration/test_execd/test_run_active_response/test_firewall_drop.py]: Applied on commit 51071c4c7601f7a486736ec2ad72f62d52673652

File: tests/integration/test_execd/test_run_active_response/test_restart_wazuh.py (hunk 2 above)
Comment (QU3B1M) [reply tests/integration/test_execd/test_run_active_response/test_restart_wazuh.py]: Applied on commit 51071c4c7601f7a486736ec2ad72f62d52673652

File: tests/integration/test_execd/test_run_active_response/conftest.py
Comment (QU3B1M) [reply tests/integration/test_execd/test_run_active_response/conftest.py:1]: Moved to the general `execd` folder on commit 51071c4c7601f7a486736ec2ad72f62d52673652

**Review by TomasTurina — CHANGES_REQUESTED** [review]
Rebase again.

**Comment by QU3B1M** [comment]
> Rebase again.

Done it! 🚀

**Review by TomasTurina — APPROVED** [review]

**Pull request merged**